python_modules/dagster/dagster_tests/definitions_tests/test_definitions_class.py

```diff
@@ -1098,3 +1098,26 @@ def check1():
     check_key = AssetCheckKey(AssetKey("asset1"), "check1")
     assert defs.get_asset_graph().asset_check_keys == {check_key}
     assert check_key in defs.get_repository_def().asset_checks_defs_by_key
+
+
+def test_map_asset_specs() -> None:
+    specs = [AssetSpec("asset1"), AssetSpec("asset2")]
+    defs = Definitions(assets=specs)
+    spec_lambda = lambda spec: spec.merge_attributes(tags={"foo": "bar"})
+    mapped_defs = defs.map_asset_specs(func=spec_lambda)
+    # seems like group_name "default" gets auto applied here? we should investigate that, feels strange but not immediately harmful.
```

Comment:
I need to delete that. It was because I was running get_all_asset_specs() instead of checking assertion on .assets initially.